### DISCUSSION TITLE
Improvements to grafana annotations api

### DISCUSSION THREAD

**Discussion by pedrooct** [grafana/grafana#38697] [General]
Hi.

I would like to know if there is new plans for the annotations api. It seems lacking on features, for exemple:

- Allow delete a batch of annotations by tag;
- Allow for search by match any;

Thank you for the awesome work.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
